src/network/networkd-address-pool.c

```diff
@@ -76,15 +69,15 @@ int address_pool_setup_default(Manager *m) {
         if (r < 0)
                 return r;
 
-        r = address_pool_new_from_string(m, AF_INET, "10.0.0.0", 8);
+        r = address_pool_new_from_string(m, AF_INET, "192.168.0.0", 16);
         if (r < 0)
                 return r;
 
         r = address_pool_new_from_string(m, AF_INET, "172.16.0.0", 12);
         if (r < 0)
                 return r;
 
-        r = address_pool_new_from_string(m, AF_INET, "192.168.0.0", 16);
+        r = address_pool_new_from_string(m, AF_INET, "10.0.0.0", 8);
```

Comment:
I think those additions were sorted by mask size previously... Not that it matters.

Reply:
Previously, the new entry was prepended to the LIST. But now it is managed by OrderedSet, and it always append the new entry. So, we need to invert the order.